statsmodels/distributions/copula/archimedean.py

```diff
@@ -72,12 +77,12 @@ def logpdf(self, u, args=()):
         return logpdfv
 
 
-class FrankCopula(ArchimedeanCopula):
+class FrankCopulaGeneratorBased(ArchimedeanCopula):
```

Comment:
No need to rename.
All archimedean copulas need to call super or add the transform.
We need the transform in the classes to have some things work generically

Reply:
Ah ok I understand! I will merge both. And add the transform call to every class.